Source/ASDisplayNode.mm

```diff
@@ -2260,6 +2261,15 @@ - (NSArray *)subnodes
   return _cachedSubnodes ?: @[];
 }
 
+- (ASDisplayTree *)_getOrCreateDisplayTreeIfNeeded
```

Comment:
Naming nit how about _displayTreeCreateIfNeeded ? Arguably the name you have here is a bit more explicit, but I don't think most of the framework is using "get" in method names.